plugins/GSdx/GSDrawScanlineCodeGenerator.x86.cpp

```diff
@@ -2889,7 +2889,7 @@ void GSDrawScanlineCodeGenerator::ReadTexel_SSE(int pixels, int mip_offset)
 				movdqa(ptr[&m_local.temp.test], xmm7);
 			}
 
-			for(int j = 0; j < 4; j++)
+			for(uint8 j = 0; j < 4; j++)
```

Comment:
In theory indexing by natural word sized variables should be fastest but yeah, the difference is most likely negligible or subject to a compiler optimization. This loop may even end up being unrolled.

There is also uint8_fast_t if you want to make sure!